src/components/list/_list.scss

```diff
@@ -103,15 +106,13 @@ $include-html: false !default;
     @include mint-list-basic-styles();
     display: inline-block;
     vertical-align: inherit;
-    font-size: $breadcrumbListFontSize;
-    line-height: $breadcrumbListFontSize * 1.5;
     color: $breadcrumbListSeparatorColor;
 
     &__element {
       display: inline-block;
 
       &:after {
-        content: '·';
+        content: '•';
```

Comment:
I think that Asia used wrong dot in these HTML based mockups, a different dot is being used on mockups from PhotoShop. But I'll check it on Monday and, if I'm right, I'll fix it in a separate PR. Meanwhile you can merge it.